src/gui/gui_common_elements.hpp

```diff
@@ -237,25 +227,6 @@ class province_liferating_progress_bar : public standard_province_progress_bar {
 	float get_progress(sys::state& state, dcon::province_id prov_id) noexcept override {
 		return state.world.province_get_life_rating(prov_id) / 100.f;
 	}
-
-	tooltip_behavior has_tooltip(sys::state& state) noexcept override {
-		return tooltip_behavior::variable_tooltip;
-	}
-
-	void update_tooltip(sys::state& state, int32_t x, int32_t y, text::columnar_layout& contents) noexcept override {
```

Comment:
Without this tooltip, how do we determine what the numerical liferating of a province is?

Reply:
gui/gui_province_window.hpp:17
it was moved into its own class

Reply:
Then what is this class used for? As far as I can remember, the liferating bar only appears in the province window

Reply:
its meant to cover both potential and current scenario, z.B in future if we have a need to show two life rating bars, or if a mod does so then its covered by this. Otherwise it just keeps consistency with the other classes.